account/views.py

```diff
@@ -0,0 +1,291 @@
+from flask import (
+    Blueprint,
+    flash,
+    redirect,
+    render_template,
+    request,
+    url_for,
+)
+from flask_login import (
+    current_user,
+    login_required,
+    login_user,
+    logout_user,
+)
+from flask_rq import get_queue
+
+from app import db
+from app.account.forms import (
+    ChangeEmailForm,
+    ChangePasswordForm,
+    CreatePasswordForm,
+    LoginForm,
+    RegistrationForm,
+    RequestResetPasswordForm,
+    ResetPasswordForm,
+)
+from app.email import send_email
+from app.models import User
+
+account = Blueprint('account', __name__)
+
+
+@account.route('/login', methods=['GET', 'POST'])
+def login():
+    """Log in an existing user."""
+    form = LoginForm()
+    if form.validate_on_submit():
+        user = User.query.filter_by(email=form.email.data).first()
+        if user is not None and user.password_hash is not None and \
+                user.verify_password(form.password.data):
+            login_user(user, form.remember_me.data)
+            flash('You are now logged in. Welcome back!', 'success')
+            return redirect(request.args.get('next') or url_for('main.index'))
+        else:
+            flash('Invalid email or password.', 'error')
+    return render_template('account/login.html', form=form)
```

Comment:
**Suggestion:** The login() function could be improved by separating the validation logic from the routing logic. This would make the code more modular and easier to test.

```suggestion
def validate_login(form):
    user = User.query.filter_by(email=form.email.data).first()
    if user is not None and user.password_hash is not None and \
            user.verify_password(form.password.data):
        return user
    else:
        return None

@account.route('/login', methods=['GET', 'POST'])
def login():
    """Log in an existing user."""
    form = LoginForm()
    if form.validate_on_submit():
        user = validate_login(form)
        if user:
            login_user(user, form.remember_me.data)
            flash('You are now logged in. Welcome back!', 'success')
            return redirect(request.args.get('next') or url_for('main.index'))
        else:
            flash('Invalid email or password.', 'error')
    return render_template('account/login.html', form=form)

```